### ISSUE TITLE
Error for some unknown reason

### ISSUE THREAD

**Issue by sahanatroam** [suchipi/transform-imports#5]
```
/Users/sahandissanayake/frontend/src/components/charts/HistoryBarChart.js
{ Error: Line 26: Invalid regular expression: missing /
    at ErrorHandler.constructError (/Users/sahandissanayake/absolute-import-transport/node_modules/esprima/dist/esprima.js:5012:22)
    at ErrorHandler.createError (/Users/sahandissanayake/absolute-import-transport/node_modules/esprima/dist/esprima.js:5028:27)
    at ErrorHandler.throwError (/Users/sahandissanayake/absolute-import-transport/node_modules/esprima/dist/esprima.js:5035:21)
    at Scanner.throwUnexpectedToken (/Users/sahandissanayake/absolute-import-transport/node_modules/esprima/dist/esprima.js:5164:35)
    at Scanner.scanRegExpBody (/Users/sahandissanayake/absolute-import-transport/node_modules/esprima/dist/esprima.js:6139:23)
    at Scanner.scanRegExp (/Users/sahandissanayake/absolute-import-transport/node_modules/esprima/dist/esprima.js:6204:29)
    at Tokenizer.getNextToken (/Users/sahandissanayake/absolute-import-transport/node_modules/esprima/dist/esprima.js:6675:56)
    at Object.tokenize (/Users/sahandissanayake/absolute-import-transport/node_modules/esprima/dist/esprima.js:154:36)
    at Object.parse (/Users/sahandissanayake/absolute-import-transport/node_modules/recast/lib/parser.js:48:30)
    at transformImports (/Users/sahandissanayake/absolute-import-transport/node_modules/transform-imports/dist/index.js:31:20)
  index: 1125,
  lineNumber: 26,
  description: 'Invalid regular expression: missing /' }
```

*Content of HistoryBarChart.js*
```
/* eslint-disable react/prop-types */

import PropTypes from 'prop-types'
import React from 'react'
import { CustomToolTipContent } from './CustomToolTip'

import {BarChart, CartesianGrid, Tooltip, Bar, ResponsiveContainer, XAxis, YAxis} from 'recharts'


class HistoryBarChart extends React.Component {

    render() {
        const {dataProvider, barKey, tickLine, xAxisLine, yAxisLine, xAxisKey, tickFormatter} = this.props

        return (

            <ResponsiveContainer height={220}>
                <BarChart
                    data={dataProvider}
                    margin={{top: 5, bottom: 5}}>
                    <XAxis tickLine={tickLine} axisLine={yAxisLine} dataKey={xAxisKey} />
                    <YAxis tickLine={tickLine} axisLine={xAxisLine} tickFormatter={tickFormatter} />
                    <CartesianGrid vertical={false} />
                    <Tooltip cursor={false} animationDuration={0} content={<CustomToolTipContent formatter={this.props.formatter} unhandled />} />
                    <Bar dataKey={barKey} fillOpacity={0.4} fill="#4b87ff" stroke="#4b87ff" />
                </BarChart>
            </ResponsiveContainer>
        )
    }
}


HistoryBarChart.displayName = 'History Bar Chart'

HistoryBarChart.propTypes = {
    barKey: PropTypes.string,
    xAxisKey: PropTypes.string,
    xAxisLine: PropTypes.bool,
    yAxisLine: PropTypes.bool,
    tickLine: PropTypes.bool,
    tickFormatter: PropTypes.func,
    formatter: PropTypes.func,
    dataProvider: PropTypes.array
}

HistoryBarChart.defaultProps = {
    formatter: (x) => { return x },
    xAxisLine: true,
    yAxisLine: true,
    tickLine: true,
}

HistoryBarChart.contextTypes = {
    muiTheme: PropTypes.object
}

export default HistoryBarChart
```

**Comment by dvsekhvalnov** [comment]
@suchipi any news on this one? Faced same when parsing .jsx files. 

Or at least may be some ideas what to look for?

**Comment by dvsekhvalnov** [comment]
Here we go #6 

Root cause, babel parser was not configured to return token which causes `recast` to fallback to esprima that not handles jsx well (https://github.com/benjamn/recast/blob/3594aa3c32f8a09ae1879b1e828bd24b637fbefa/lib/parser.ts#L40)

**Comment by suchipi** [comment]
Fixed thanks to @dvsekhvalnov in version 2.1.1
